### PR TITLE
Document how to install

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Keeping the trash in your ./vendor dir to a minimum.
 
 Make sure you're using go1.6 or later version.
 
- 0. Download and extract `trash` to your PATH
+ 1. Download and extract `trash` to your PATH: `go get github.com/rancher/trash`.
```
Comment (imikushin) [inline README.md]: `go get` totally works, but there are actual binary releases to download.

So, maybe also add a link to https://github.com/rancher/trash/releases ?

Comment (imikushin) [inline README.md]: Also, can you preserve the list numbering?

File: README.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ Keeping the trash in your ./vendor dir to a minimum.
...
  1. Copy `vendor.conf` file to your project and edit to your needs.
- 2. Run `trash`
+ 3. Run `trash`.
```
Comment (imikushin) [inline README.md]: Seriously? 
Okay, just kidding: the dot is fine 😸

Comment (imikushin) [inline README.md]: But leave alone the numbering!

File: README.md (hunk 1 above)
Comment (AlekSi) [inline README.md]: Maybe. You know better. :)

Comment (joshwget) [inline README.md]: It's currently numbered starting at 0, so it seems they both render the same. @imikushin most markdown I've seen begins at 1. Any reason why we can't just switch to that?

**Comment by AlekSi** [comment]
PTAL

**Pull request merged**